generators/docker-base.js

```diff
@@ -49,7 +49,7 @@ function checkImages() {
             runCommand = './mvnw package -Pprod verify jib:dockerBuild';
```

Comment:
Related, `package` is no longer required for the Jib standard (non-WAR) containerization. Normally, `./mvnw compile jib:dockerBuild` should work (although if we were to do the WAR containerization, `package` is required.)